src/server/graphql/httpGraphQLHandler.js

```diff
@@ -1,6 +1,6 @@
 import {graphql} from 'graphql';
 import RethinkDataLoader from 'server/utils/RethinkDataLoader';
-import IntranetSchema from './intranetSchema';
+import IntranetSchema from 'server/graphql/intranetSchema/intranetSchema';
```

Comment:
relative imports are the devil! files can move, and when they do, i don't want to see a series of `../../../..foo`. It's ugly & leaves you with errors like referencing /server/utils instead of /server/graphql/utils.

that said, this one was done automatically. i have my editor set up to auto-import modules as i move or use them & to use absolute paths for everything.

Reply:
Haha `../../../../` is always funny. I tend to prefer flatter source trees whenever possible for this reason :)